Test failures with Python 3.13

Python 3.13 should be released this week so we're testing more packages with it in Gentoo in preparation.

nml's testsuite passes with Python 3.12 but seems to fail for me with Python 3.13:
```
$ make regression NML_FLAGS='-s -c --verbosity=1 --no-cache' ; echo $?
/usr/bin/env python3 setup.py build_ext --inplace
running build_ext
copying build/lib.linux-x86_64-cpython-313/nml_lz77.cpython-313-x86_64-linux-gnu.so ->
make -C regression
make[1]: Entering directory '/tmp/nml/regression'
Running test 001_action8
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: Entering directory '/tmp/nml/regression'
Running test 002_sounds
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: Entering directory '/tmp/nml/regression'
[...]
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: Entering directory '/tmp/nml/regression'
Illegal character '     ' (character code 0x09) at "nml_output/001_action8.nml", line 2, column 1
make[1]: *** [Makefile:19: 001_action8] Error 1
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: *** Waiting for unfinished jobs....
make[1]: Entering directory '/tmp/nml/regression'
Illegal character '     ' (character code 0x09) at "nml_output/022_disable_item.nml", line 2, column 1
make[1]: *** [Makefile:19: 022_disable_item] Error 1
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: Entering directory '/tmp/nml/regression'
Illegal character '     ' (character code 0x09) at "nml_output/025_loop.nml", line 4, column 1
make[1]: *** [Makefile:25: 025_loop] Error 1
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: Entering directory '/tmp/nml/regression'
Illegal character '     ' (character code 0x09) at "nml_output/011_snowline.nml", line 2, column 1
make[1]: *** [Makefile:25: 011_snowline] Error 1
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: Entering directory '/tmp/nml/regression'
Illegal character '     ' (character code 0x09) at "nml_output/027_airport_layout.nml", line 2, column 1
make[1]: *** [Makefile:25: 027_airport_layout] Error 1
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: Entering directory '/tmp/nml/regression'
Illegal character '     ' (character code 0x09) at "nml_output/018_airport_tile.nml", line 2, column 1
make[1]: *** [Makefile:19: 018_airport_tile] Error 1
make[1]: Leaving directory '/tmp/nml/regression'
make[1]: Entering directory '/tmp/nml/regression'
```

--- a/nml/tokens.py
+++ b/nml/tokens.py
@@ -195,7 +195,7 @@ class NMLLexer:
         self.increment_lines(t.value.count("\n"))
 
     def t_ignore_whitespace(self, t):
-        "[ \t\r]+"
+        r"[ \t\r]+"
         pass
 
     def t_line_directive1(self, t):

